modules/core/src/main/java/org/apache/fluo/core/impl/TransactionImpl.java

```diff
@@ -895,6 +904,22 @@ public int getSize() {
     return size;
   }
 
+  private <V> void addCallback(CompletableFuture<V> cfuture, CommitData cd,
+      OnSuccessInterface onSuccessInterface) {
```

Comment:
Should this have a type parameter like `OnSuccessInterface<V>`?

Reply:
Thanks, the casting was really bothering me and I knew it was from something stupid/small.